kafka-rest/src/test/resources/log4j.properties

```diff
@@ -6,6 +6,9 @@ log4j.appender.stdout.layout.ConversionPattern=[%d] %p %m (%c:%L)%n
 
 log4j.logger.kafka=ERROR
 log4j.logger.org.apache.kafka=ERROR
+log4j.logger.org.apache.kafka.clients.producer=TRACE
+log4j.logger.org.apache.kafka.clients.consumer=TRACE
```

Comment:
nit: I'm a bit concerned that `TRACE` will create an unmanageable amount of stuff in the logs, but that's entirely based on previous experience from different systems, I have no idea what is the verbosity difference between `TRACE` and `DEBUG` and whether `DEBUG` even contains what you need. If it does though, I'd suggest logging at that, not `TRACE`.

Reply:
I've taken this out, the rest of the unit tests log at error, so we'll stick with that.

But, I think it makes it virtually impossible to debug failures that happen on jenkins, so once our tests are a bit more robust, and we've cleared up all the easy to reproduce locally failures, we should consider flipping this to be TRACE for the clients and for REST so we have some chance of debugging the hard failures too.